src/Microsoft.ML.Data/Evaluators/MultiOutputRegressionEvaluator.cs

```diff
@@ -176,11 +178,11 @@ public sealed class Counters
 
                 private readonly IRegressionLoss _lossFunction;
 
-                public Double L1 { get { return _sumWeights > 0 ? _sumL1 / _sumWeights : 0; } }
+                public Double L1 => _sumWeights > 0 ? _sumL1 / _sumWeights : 0;
```

Comment:
Huh interesting... I might want to start using this thing.

Reply:
i did that by hand, on the first iteration.

Reply:
Oh. I got pretty excited there that it was this capable for a second. 😄